terraform/modules/front_door/cloudfront.tf

```diff
@@ -59,6 +60,12 @@ resource "aws_cloudfront_distribution" "this" {
     }
   }
 
+  logging_config {
+    bucket          = aws_s3_bucket.cloudfront_access_logs.bucket_domain_name
+    include_cookies = false
+    prefix          = ""
```

Comment:
`prefix` is optional according to the terraform docs - do we need this line?

Reply:
For our `tf_state_backend` terraform module and `aws_s3_bucket_logging` resources, we had to provide similar prefixes with an empty string 😅, so I wanted to be consistent and explicit here as well, with the benefit that it might be clear the logs live in the root of the bucket